.github/workflows/codespell-private.yml

```diff
@@ -56,4 +56,6 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Flake8 with annotations
-        uses: TrueBrain/actions-flake8@v1.2
+        uses: TrueBrain/actions-flake8@v2.1
+        with:
+          flake8_version: 3.9.2
```

Comment:
What's your reason for pinning it? Is it running with an older installed version otherwise? Maybe a pip upgrade or something could be used (or equivalent pushed back to the action)?

Reply:
I would rather use:
`uses: TrueBrain/actions-flake8@v2`
instead of:
`uses: TrueBrain/actions-flake8@v2.1`

That would run the latest 2.* version, whatever it is. There **is** a risk an upgrade of the flake8 action might break things. Freezing everything does alleviate the risk. I doubt there will be many breakages, if any. Most projects use `@v2`.

Reply:
it'll pick the 'latest' version of flake8 if we don't pin the flake8 version; that could change underneath us and cause surprises.  pinning means you never have that issue; you'd only update when you explicitly update in a PR.  (note the previous version of the action was pinned to a specific version of flake8; it just wasn't visible or configurable)

I'd prefer to pin everything, including the action version, for this same reason.

Reply:
Personally I'd favour it breaking on CI, rather than someone trying to contribute for the first time and finding a whole load of new errors when they happen to run with a newer flake8, but that's just me, I can see the logic for feeling differently too.

Reply:
Ok it's not a big deal I'll make those suggested changes.

Reply:
Thanks @bwitt . We can always review in say six months and change it if we keep getting bombarded.